dashboard/src/pages/TreeDetails/Tabs/LogSheet.tsx

```diff
@@ -112,6 +118,32 @@ const LogFilesTable = ({
 
 const MemoizedLogFilesTable = memo(LogFilesTable);
 
+const TooltipWrapper = ({
```

Comment:
nit: This component basic has no logic so it is not really useful, IMO we can remove it